### ISSUE TITLE
[Scala]: Drop finch

### ISSUE THREAD

**Issue by whiplash** [the-benchmarker/web-frameworks#6872]
Finch is not being maintained that much. There are ongoing discussions on its future. I think it's fine if we drop it (for now).

**Comment by waghanza** [comment]
could you point me on this discussion ?

**Comment by whiplash** [comment]
https://github.com/finagle/finch/issues/1633

**Comment by whiplash** [comment]
no. the opposite. finch uses finagle. http://twitter.github.io/finagle/

**Comment by whiplash** [comment]
also https://github.com/finagle/finch/issues/1396 

might be worth poking on @arron-green and/or @rpless

**Comment by rpless** [comment]
Finch has had releases a couple times since that issue was closed, but with Finagle development mostly drying up after the twitter acquisition, I'm not really sure what the future holds for it. Totally understand if you don't want the maintenance burden.

**Comment by waghanza** [comment]
for me if finch is used in production, and could be adopted for new projects, it has it place here @whiplash 

however, not to add confusion, if it is not recommended to use finch in production anymore, I think we should remove it
